examples/siro_sandbox/controllers/fetch_baselines_controller.py

```diff
@@ -411,10 +417,37 @@ def act(self, obs, env):
         human_pos = env._sim.agents_mgr[
             1 - self._agent_idx
         ].articulated_agent.base_transformation.translation
+        human_rot = env._sim.agents_mgr[
+            1 - self._agent_idx
+        ].articulated_agent.base_rot
         robot_pos = env._sim.agents_mgr[
             self._agent_idx
         ].articulated_agent.base_transformation.translation
 
+        # Check if the human walks or rotates
```

Comment:
You should move all this logic into a helper function for readability.

Reply:
Thanks, Eric! Agree with you!